content/docs/getting-started/connect-dbaas.md

```diff
@@ -64,3 +65,25 @@ Configure `cURL` with DBaaS credentials.
 ```sh
 alias curl="curl -s -k -u \"${NUODB_CP_USER}:${NUODB_CP_PASSWORD}\""
 ```
+
+### Setting up the Terraform Provider
+
+To use terraform to manage your databases, you will want to use the [`nuodbaas` provider](https://registry.terraform.io/providers/nuodb/nuodbaas).
+The [provider documentation](https://registry.terraform.io/providers/nuodb/nuodbaas/latest/docs#schema) covers all of the attributes you can define but, if they are not set, the provider can infer a configuration using the environment variables set above.
+
+```terraform
+terraform {
+  required_providers {
+    nuodbaas = {
+      source  = "registry.terraform.io/nuodb/nuodbaas"
+      version = "1.2.0"
+    }
+  }
+}
+
+provider "nuodbaas" {
+    # The provider does not support self signed certificates.

```

Comment:
We don't need to call this out as a limitation. The line below is sufficient to explain why this is necessary.

Reply:
I also reordered the second sentence since it is carrying more meaning now.